projects/storefrontlib/src/lib/my-account/order/order-details/order-details.component.ts

```diff
@@ -1,11 +1,8 @@
 import { Component, OnInit, OnDestroy, ViewEncapsulation } from '@angular/core';
-
-import { AuthService, RoutingService } from '@spartacus/core';
-
 import { Observable, Subscription, combineLatest } from 'rxjs';
 import { map } from 'rxjs/operators';
-
 import { UserService } from '../../../user/facade/user.service';
+import { AuthService, RoutingService, Order } from '@spartacus/core';
 import { Card } from '../../../ui/components/card/card.component';
 
 @Component({
```

Comment:
Can `potentialProductPromotions` on line 83 have a type?

Reply:
I don't see any instance of potentialProductPromotions in this file

Reply:
`Product` has `potentialPromotions` field